### DISCUSSION TITLE
Do you use Karabiner.ts also for Text expansion?

### DISCUSSION THREAD

**Discussion by mseibert** [evan-liu/karabiner.ts#106] [Q&A]
I am asking myself if this solution is good for use cases that my colleagues solve with TextExpander. Would you recommend it?

**Comment by evan-liu** [comment]
I am not sure what use cases your colleagues are solving with TextExpander. I use it for [mostly often used code snippets](https://github.com/evan-liu/karabiner-config/blob/36953ca6b113450e493b82c96fa3b8cabc616374/src/layers/emoji.ts#L40C2-L40C2) like `map('l').toTypeSequence('console.log()←')`, but I do not used it for snippets that are not used often (I use [Raycast](https://www.raycast.com/) snippets for them instead)
